src/bindings.cpp

```diff
@@ -1,6 +1,7 @@
-#include "fast_marching_method.hpp" // BindFastMarchingMethodModule
 #include <pybind11/numpy.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
 
+#include "fast_marching_method.cpp" // BindFastMarchingMethodModule
```

Comment:
I find including cpp files a bit strange; can't the usual methods be used?

Reply:
It can but I see no need for a header file here tbh, given that this needs to be copied once and before the implementation was completely in the header file.

Reply:
ok reverted back to using headers